Description: Publish Ubuntu ddeb files to the repositories
Author: Lucas Kanashiro <lucas.kanashiro@collabora.com>
Last-Updated: 2019-03-21

--- a/src/backend/bs_publish
+++ b/src/backend/bs_publish
@@ -1507,7 +1507,7 @@ sub publish {
       if ($bin =~ /^.+-[^-]+-[^-]+\.([a-zA-Z][^\/\.\-]*)\.d?rpm$/) {
 	$p = "$1/$bin";
 	$p = $1 eq 'src' || $1 eq 'nosrc' ? "SRPMS/$bin" : "RPMS/$bin" if $repotype{'resarchhack'};
-      } elsif ($bin =~ /^.+_[^_]+_([^_\.]+)\.u+deb$/) {
+      } elsif ($bin =~ /^.+_[^_]+_([^_\.]+)\.[ud]?deb$/) {
 	$p = "$arch/$bin";
       } elsif ($bin =~ /\.exe$/) {
 	$p = "$bin";
